content/docs/pulumi-cloud/copilot/api.md

```diff
@@ -0,0 +1,243 @@
+---
+title_tag: "Pulumi Copilot API"
+meta_desc: Learn about Pulumi Copilot REST APIs for integrating Pulumi Copilot capabilities into your applications and workplace platforms.
+title: Pulumi Copilot API
+h1: Pulumi Copilot API Overview
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+    cloud:
+        name: Pulumi Copilot API
+        parent: pulumi-cloud-copilot
+        weight: 1
+        identifier: pulumi-cloud-copilot-api
+---
+
+{{% notes "info" %}}
+Pulumi Copilot's API is currently in preview and subject to change. It is not currently suitable for production use.
+{{% /notes %}}
+
+The Pulumi Copilot REST API (currently in Preview) is part of the [Pulumi Cloud REST API](https://www.pulumi.com/docs/pulumi-cloud/reference/cloud-rest-api) that is used to integrate Pulumi Copilot capabilities with other applications or tools.
+
+## Endpoint URL
+
+For the Managed Pulumi Cloud (i.e. [app.pulumi.com](https://app.pulumi.com/)), API endpoints are prefixed with the following url:
+
+```
+https://api.pulumi.com
+```
+
+Currently, [Self-Hosted Pulumi Cloud](/docs/pulumi-cloud/self-hosted/) is not supported by the Pulumi Copilot REST API.
+
+## Authentication
+
+All requests must be authenticated using a token via the `Authorization` HTTP header.
+
+The `Authorization` header must be in the form below with the literal string `token`, then a space, then your access token value.
+
+```
+Authorization: token {token}
+```
+
+To view your access tokens, or create a new one, view the <a href="https://app.pulumi.com/account/tokens" target="_blank">Access Tokens</a> page. You will see a list of past tokens, when they were last used, and have the ability to revoke them.
+
+The Pulumi Copilot REST API will return a 401 status code if the token is missing or invalid.
+
+## Required request headers
+
+The following header is required:
+
+```
+Content-Type: application/json
+```
+
+## Start conversation
+
+Starts a new conversation with Pulumi Copilot with a query.
+
+```
+POST /api/ai/chat/preview
+```
+
+### Request format
+
+#### Request body schema
+
+```JSON
+{
+    "query": string,
+    "state": {
+        "client": {
+            "cloudContext": {
+                "orgId": string,
+                "url": string
+            }
+        }
+    }
+}
+```
+
+#### Fields description
+
+- `query`: The natural language query to process (e.g., "Who are the users in my org?").
+- `orgId`: The identifier for your Pulumi organization (e.g., "acme").
+- `url`: The URL of the resource in the Pulumi Cloud that provides context to the Pulumi Copilot (e.g., "https://app.pulumi.com/myorg/project1/dev/updates/4"). This parameter must start with "https://app.pulumi.com".
```

Comment:
Agreed! Maybe a better option would be to have a path (the string after https://app.pulumi.com) in which case it would be truly optional. What I like about the url is that it's easy to explain: it's what you see in the browser when talking to Copilot! 
I'll consider this for the next round of changes, based on feedback.